docs/Menu-Commands.md

```diff
@@ -0,0 +1,78 @@
+  | --------------------- | --------------- | ----------------------- |
+  | `zowe.<command>`      | Data Sets       | `zowe.addFavorite`      |
+  | `zowe.uss.<command>`  | USS             | `zowe.uss.addFavorite`  |
+  | `zowe.jobs.<command>` | Jobs            | `zowe.jobs.addFavorite` |
+
+- `title` property: Be sure to localize the command title's value. This can be done by following the steps below:
+
+  1. Create a concise, descriptive key for the title string and wrap it in `%` on each side. Consider prefixing the key name with `mvs.`, `uss.`, or `jobs.` to indicate if the key is for the Data Sets, USS, or Jobs view, respectively. (For example, the key `%uss.addFavorite%` is used for the `addFavorite` command in the USS view.) If the command is more related to profiles than a specific view, consider prefixing the key name with `profile.`.
+  1. in `package.nls.json`, define the user-facing string for the key. (For example: `"uss.addFavorite": "Add to Favorites"`.)
+
+  - More details on localization can be found in the [Developer's ReadMe](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/Developer's%20ReadMe.md#adding-strings).
+
+- `category` property: The value of this should be "Zowe Explorer".
+
+**Note:** By default, defined commands will appear in VS Code's Command Palette. If the command being added is not designed to work from the Command Palette, be sure to add an entry for the command in the `contributes.menus.commandPalette` section of `packages/zowe-explorer/package.json`, and specify `"when": "never"`. (For more details, see VS Code's documentation on [specifying command visibility in the Command Palette](https://code.visualstudio.com/api/references/contribution-points#Context-specific-visibility-of-Command-Palette-menu-items).)
+
+## Contributing context menu items
+
+Context menus appear when the user right-clicks on an item (for example, a data set or profile). Commands can be added into the context menus by creating an entry for a [defined command](#defining-a-new-command) in the `menus contributes.menus.view/item/context` section of `packages/zowe-explorer/package.json`.
+
+- If adding a command to a Zowe Explorer context menu, be sure to assign a value to the `group` property of the entry. This helps provide visual separation between different command categories in the UI. You can see examples of context menu groups in the current menu items in Zowe Explorer's `package.json`. Follow the steps below when assigning an entry's `group` property:
+
+  1. First, check the `package.json` to see if there is an existing context menu group that the new command can intuitively be categorized under (from an end-user's perspective).
+  1. If the new command does not fit under any existing context menu groups, create a new context menu group by assigning the `group` property a new value that follows [Zowe Explorer's context menu group naming convention](#zowe-explorer-context-menu-group-naming-convention).
```

Comment:
Would still shorten this: `Zowe Explorer's context menu group naming convention`

to this: `our menu group naming conventions`

Reply:
I have an aversion to writing in first-person for documentation. Maybe we can just say: `the menu group naming conventions`. It's the same number of letters as your suggestion. :)

Reply:
I thought about this some more, and ended up going with "Zowe Explorer's menu group naming conventions".

This option avoids first-person, but also isn't as generic as just saying "the menu group naming conventions".